.github/workflows/workflow.yml

```diff
@@ -28,7 +32,7 @@ on:
 
 jobs:
   inventory:
-    runs-on: ubuntu-latest
+    runs-on: ${inputs.runs-on}
```

Comment:
FYI the syntax is generally `${{ ... }}`. I imagine it's tolerating but worth being consistent.